specs/destination.go

```diff
@@ -19,9 +19,9 @@ type Destination struct {
 }
 
 const (
-	WriteModeAppend WriteMode = iota
+	WriteModeOverwriteDeletestale WriteMode = iota
 	WriteModeOverwrite
-	WriteModeOverwriteDeletestale
+	WriteModeAppend
 )
 
 func (d *Destination) SetDefaults() {
```

Comment:
Github doesn't allow me to comment on that line, but on [line 68](https://github.com/cloudquery/plugin-sdk/pull/224/files#diff-7c146e9fc20b6cf14b74be16d0cece21f950b510aa75b4d92c9527cfd1f7a8f1R68), let's change it to `overwrite-delete-stale` (with a dash)

Reply:
done